### PR TITLE
Fixed Menu Music playing during Intro video due to a late load.

### PR DESCRIPTION
**Pull request by xesf**

### CODE REVIEW

File: src/ui/GameUI.tsx [hunk 1]
```diff
@@ -256,7 +256,11 @@ export default class GameUI extends FrameListener<GameUIProps, GameUIState> {
     showMenu(inGameMenu = false) {
         this.state.game.pause();
         const audioMenuManager = this.state.game.getAudioMenuManager();
-        audioMenuManager.getMusicSource().loadAndPlay(6);
+        audioMenuManager.getMusicSource().load(6, () => {
+            if (this.state.showMenu) {
```
Comment (slwilliams) [inline src/ui/GameUI.tsx:260]: Perhaps add a comment explaining why this is needed over the simpler loadAndPlay?

E.g. "Ensure we don't start the music if it took a while to load the audio file and the user has already made it past the menu".

Comment (xesf) [reply src/ui/GameUI.tsx:260]: Cheers.
We tend to reduce the amount of comments in the code, so we can have everything more explicit as possible. Although, I will be refactoring the whole Audio part and improve this piece soon.

Comment (xesf) [reply src/ui/GameUI.tsx:261]: Pushed change with the comment.

**Comment by xesf** [comment]
Closing PR as I have another one in-progress.

**Comment by agrande** [comment]
@xesf can I delete the branch then, or do you prefer to keep it opened for some time?

**Comment by xesf** [comment]
@agrande That's done. I have removed the branch. Thanks for the heads-up.

**Pull request closed without merging**